_artifacts/toc.yml

```diff
@@ -0,0 +1,2714 @@
+main-menu:
```

Comment:
Can you confirm this whole file should be added like this? I am not entirely certain what changes you made here as the file already exists. A rebase of your PR may fix it.